#!/usr/bin/env python
PACKAGE = "masterslave"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("motionScaling", double_t, 0, "Motion Scaling Factor", 1, 0.01, 2)
gen.add("rotationScaling", double_t, 0, "Rotation Scaling Factor", 1, 0.01,2)


exit(gen.generate(PACKAGE, "MasterSlaveManipulationAbsolute", "MasterSlaveManipulationAbsolute"))
